### PR TITLE
Forum datastore logics

### PR DESCRIPTION
**Pull request by Byambaa0325**
Basic Datastore Logic to Retrieve and Store a Forum

### CODE REVIEW

File: src/main/java/com/google/codeu/data/Datastore.java [hunk 1]
```diff
@@ -318,4 +321,40 @@ public void storeArticle(Article article) {
...
+   * @param id id of a forum
+   * @return Forum object
+   * @throws EntityNotFoundException when no such forum with id is found
+   */
+  public Forum getForum(String id) throws EntityNotFoundException {
+    Key key = KeyFactory.createKey("Forum", id);
+    Entity forumEntity = datastore.get(key);
+    UUID uuid = UUID.fromString(id);
+    String title = (String) forumEntity.getProperty("title");
+    List<User> owners = (List<User>) forumEntity.getProperty("owners");
```
Comment (tugushka) [inline src/main/java/com/google/codeu/data/Datastore.java]: Oh wait, can you do this?

**Review by tugushka — APPROVED** [review]

**Comment by tugushka** [comment]
By the way, you're indenting with 4 spaces on Forum.java

File: src/main/java/com/google/codeu/data/Datastore.java (hunk 1 above)
Comment (Byambaa0325) [reply src/main/java/com/google/codeu/data/Datastore.java]: Not really used it as a placeholder of sorts but I changed it in the future commits.

**Comment by Byambaa0325** [comment]
Thanks, fixing the indent before merging

**Pull request merged**